code-name-taran/src/com/spaghettisoft/component/hangman/Player.java

```diff
@@ -0,0 +1,81 @@
+package com.spaghettisoft.component.hangman;
+
+import com.spaghettisoft.globals.StaticObjects;
+
+public class Player {
+
+	private String name;
+	private static final char LIFE = '@';
+	private static final char DEAD = 'X';
+	private static final int NUMBER_OF_LIFES = 6;
+	private int lifeCounter;
+	private char[] lifes = new char[NUMBER_OF_LIFES];
+	private boolean myTurn = false;

```

Comment:
You are using this  variable to check whether the player is on turn, but the only place where you are reading it is inside the game logic class. I suggest you to create a reference "playerOnTurn" inside the Game Logic class, which keeps the current player on turn, and change this reference when it is needed to player1 or player2.